client/src/app/site/pages/meetings/pages/mediafiles/view-models/view-mediafile.ts

```diff
@@ -25,6 +26,18 @@ export class ViewMediafile extends BaseProjectableViewModel<Mediafile> {
         return this._model;
     }
 
+    public get inherited_access_group_ids(): Id[] {
+        return this.getMeetingMediafile()?.inherited_access_group_ids;
+    }
+
+    public get inherited_access_groups(): ViewGroup[] {
+        return this.getMeetingMediafile()?.inherited_access_groups;
+    }
+
+    public get has_inherited_access_groups(): boolean {
+        return !!this.getMeetingMediafile()?.inherited_access_group_ids.length;
+    }
```

Comment:
Should be calculated from this classes `inherited_access_group_ids` getter

Reply:
:cockroach: